code-highlighter.html

```diff
@@ -225,18 +225,21 @@
       _splitHunk: function (e) {
         var path = this._getComposedPath(e)[0];
         var lineNumber = Number(path.getAttribute('index')) + 1;
-        if (path.localName === 'iron-icon' && lineNumber !== this._amountLines) {
-          var newElem = {};
-          var x = this.code.split('\n');
-          newElem.code = x.slice(lineNumber, this.code.length - 1).join('\n');
-          newElem.beforeStart = path.getAttribute('before');
-          newElem.afterStart = path.getAttribute('after');
-          newElem.index = this.index;
-          this.code = x.slice(0, lineNumber).join('\n');
-          this.fire('add-hunk', newElem);
+        if (path.localName === 'iron-icon') {
+          if (lineNumber !== this._amountLines) {
+            var newElem = {};
+            var x = this.code.split('\n');
+            newElem.code = x.slice(lineNumber, this.code.length - 1).join('\n');
+            newElem.beforeStart = path.getAttribute('before');
+            newElem.afterStart = path.getAttribute('after');
+            newElem.index = this.index;
+            this.code = x.slice(0, lineNumber).join('\n');
+            this.fire('add-hunk', newElem);
```

Comment:
I'd like to propose renaming this to `split-code` since this event does not add a hunk at all. It just splits the highlight in two. When adding comments, the amount of hunks remain the same.